fs/opener/registry.py

```diff
@@ -282,10 +283,18 @@ def manage_fs(
         """
         from ..base import FS
 
+        def assert_writeable(fs):
+            if fs.getmeta().get("read_only", True):
+                raise ResourceReadOnly(path=str(fs_url))
```

Comment:
As it's the FS itself that is read_only, I wonder if the appropriate `path` to pass would be `/` rather than the `fs_url` ?

Reply:
This way results in the nicer error message:

```
fs.errors.ResourceReadOnly: resource '<osfs '/Users/tf/Desktop'>(read-only)' is read only
```

Reply:
I wasn't sure if people catching the error from `ResourceReadOnly` might always expect `path` to be a valid pyfs-path? :shrug: 
(I guess _theoretically_ there could be a different `FilesystemReadOnly` error, if we don't know which particular resource was being accessed)

Reply:
Ok, better to be consistent - changed it 👍